File: src/examples/syntax-examples.go
package main

import "fmt"

func main() {
    // 切片
    numbers := []int{1, 2, 3, 4, 5}
    
    // range 循环
    for i, num := range numbers {
        fmt.Printf("索引:%d 值:%d\n", i, num)
    }
    
    // 结构体
    type Person struct {
        Name string
        Age  int
    }
} 

File: src/examples/syntax-examples.py
# Python 示例
def fibonacci(n):
    if n <= 1:
        return n
    return fibonacci(n-1) + fibonacci(n-2)

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age
    
    def greet(self):
        print(f"Hello, my name is {self.name}")

# 列表推导式
squares = [x**2 for x in range(10)] 